public/401.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Error(401)</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <style>
```

Comment:
Lets leave this styles as is, but later we will move them ito separate file

Reply:
okay